lib/matplotlib/backends/backend_qt5.py

```diff
@@ -442,6 +464,52 @@ def closeEvent(self, event):
         QtWidgets.QMainWindow.closeEvent(self, event)
 
 
+class Window(WindowBase, MainWindow):
+    def __init__(self, title):
+        WindowBase.__init__(self, title)
```

Comment:
Ahh, thanks for the heads-up.  I think see the problem.  It works fine so long as we don't extend ``WindowQt5`` and then super will break... just found the perfect solution to this, just returned to the article I began reading the other day, and they have the solution.

https://rhettinger.wordpress.com/2011/05/26/super-considered-super/ (see How to Incorporate a Non-cooperative Class).  As far as I can see, we can apply the adapter patten directly in ``qt_compat.py``.

Anyway, will look into this deeper once we have the main MEP27 branch merged and turn our focus onto the individual backends.